ui-lib.css

```diff
@@ -511,10 +511,18 @@ header .logo {
   margin-right: 5px;
   vertical-align: middle;
 }
+
+.acc-pane > h3:focus {
+  outline: none;
+}
+
+.acc-pane > h3:focus,
```

Comment:
Added focus in multiple places to indicate current place for keyboard navigation, since the `outline: none;` addition above removes the default browser indicator.